src/com/nutrons/nu17/subsystems/Climber.java

```diff
@@ -0,0 +1,43 @@
+package com.nutrons.nu17.subsystems;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * @author Kevin Henriquez
+ */
+public class Climber extends Subsystem {
+	public DigitalInput MicroSwitch = new DigitalInput(0);
+
+	private Talon Climb = new Talon(3);
+
+	public void initDefaultCommand() {
+		// Set the default command for a subsystem here.
+		// setDefaultCommand(new MySpecialCommand());
```

Comment:
Don't leave code commented out unless you've got a reason, in a comment next to it. 
Otherwise its just confusing  & hard to read.